### DISCUSSION TITLE
3xx status code from action handlers

### DISCUSSION THREAD

**Discussion by KentaKudo** [hasura/graphql-engine#9433] [Question] [answered]
Hi there,

Is there a way to return 3xx status code from action handlers as returned from webhook endpoint and pass back the `location` header to let clients be redirected? TIA.

**Comment by adas98012** [comment, marked as answer]
@KentaKudo 
As mentioned in our [docs](https://hasura.io/docs/latest/actions/action-handlers/#returning-a-success-response) we support 2xx status codes (for success) and 4xx status codes (for errors).
